Lib/test/test_netrc.py

```diff
@@ -2,6 +2,8 @@
 from test import support
 from test.support import os_helper
 
+@unittest.skipIf(sys.platform == "vxworks",
```

Comment:
Would you mind adding a comment to explain that the whole test is skipped on VxWorks since this OS doesn't have the concept of home directories?

Reply:
Ensuring test_security(), test_file_not_found_in_home() and test_home_not_set() create the fake home under /tmp, VxWorks now can pass all the tests. So we will not skip the test module.